test/lsp.py

```diff
@@ -278,20 +278,20 @@ def setup_lsp(self, lsp: JsonRpcProcess, expose_project_root=True):
                     'workspaceFolders': True
                 }
             }
-        }
+        } # type: Any
         if not expose_project_root:
             params['rootUri'] = None
         lsp.call_method('initialize', params)
         lsp.send_notification('initialized')
 
     # {{{ helpers
-    def get_test_file_path(self, test_case_name):
+    def get_test_file_path(self, test_case_name: Any) -> Any:
```

Comment:
```suggestion
    def get_test_file_path(self, test_case_name: str) -> str:
```
But I could contribute that after this PR on top as well, too. hmmm (same for the other type annotation changes in this file at least)